lib/ui/view/channel/pinned_messages/pinned_messages_view.dart

```diff
@@ -51,28 +51,30 @@ class PinnedMessagesView extends StatelessWidget {
                       separatorBuilder: (context, index) => const Divider(
                           height: 24.0, indent: 56.0, thickness: 1.28),
                       itemBuilder: (context, index) => ListTile(
-                        leading:
-                            const ImageIcon(AssetImage(PinIcon), size: 24.0),
+                        leading: const ImageIcon(AssetImage(PinIcon)),
                         contentPadding: EdgeInsets.zero,
                         subtitle: Text(
                           model.pinnedMessages[index].content!,
                           style: GoogleFonts.lato(
```

Comment:
Please go to Styles and look for the text style
Use AppTextStyles instead

Reply:
Done! I've changed it to `AppTextStyles.pinnedMessageStyle`